projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -102,6 +103,40 @@ public function register_routes() {
 			)
 		);
 
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<schedule_id>[\w]+)/logs',
+			array(
+				array(
+					'methods'             => WP_REST_Server::EDITABLE,
+					'callback'            => array( $this, 'add_log' ),
+					'permission_callback' => array( $this, 'add_log_permissions_check' ),
+					'args'                => array(
+						'action'  => array(
+							'description' => 'The action to be logged',
+							'type'        => 'string',
+							'required'    => true,
```

Comment:
Would it make sense to add an `enum` parameter here that lists the accepted actions?

Reply:
You're right. I'll add it.